samples/matter/light_bulb/boards/nrf54l15dk_nrf54l15_cpuapp.overlay

```diff
@@ -26,17 +26,17 @@
 	};
 };
 
-// restore full RRAM and SRAM space - by default some parts are dedicated to FLRP
+/* restore full RRAM and SRAM space - by default some parts are dedicated to FLRP */
 &cpuapp_rram {
 	reg = <0x0 DT_SIZE_K(1524)>;
 };
 
 &cpuapp_sram {
 	reg = <0x20000000 DT_SIZE_K(256)>;
-	ranges = <0x0 0x20000000  0x40000>;
+	ranges = <0x0 0x20000000 0x40000>;
 };
 
-// TODO: re-enable HWFC once it's fixed
+/* TODO: re-enable HWFC once it's fixed */
```

Comment:
Question :)
Do we still have nrf54l15 with some issues with HWFC?

Reply:
Not sure, we can check it later.